lib/google_iap.ex

```diff
@@ -43,16 +71,19 @@ defmodule GoogleIAP do
     * `expected_date`
     * `desired_date`
   """
-  @spec defer_subscription(String.t, String.t, String.t, DateTime.t, DateTime.t) :: HTTPoison.Response
+  @spec defer_subscription(String.t, String.t, String.t, DateTime.t, DateTime.t) :: {:ok, DateTime.t | :error, String.t}
   def defer_subscription(package_name, subscription_id, token, expected_date, desired_date) do
     body = %{
       deferralInfo: %{
         expectedExpiryTimeMillis: DateTime.to_unix(expected_date),
         desiredExpiryTimeMillis: DateTime.to_unix(desired_date)
       }
     }
-    subscription_url("defer", package_name, subscription_id, token)
-    |> Client.post(body, [{"content-type", "application/json"}])
+
+    case subscription_url("defer", package_name, subscription_id, token) |> Client.post(body, [{"content-type", "application/json"}]) do
+      {:ok, response} -> {:ok, DateTime.from_unix!(response.body["newExpiryTimeMillis"])}
+      {:error, error} -> error.reason
```

Comment:
Same as above, we should probably return the idiomatic `{:ok, result}` or `{:error, reason}`